specification/cognitiveservices/AnomalyDetector/client.tsp

```diff
@@ -3,4 +3,4 @@ import "./main.tsp";
 
 using Azure.ClientGenerator.Core;
 
-@@convenientAPI(AnomalyDetector.Univariate.detectUnivariateEntireSeries, false)
+@@convenientAPI(AnomalyDetector.Univariate.detectUnivariateEntireSeries, false);
```

Comment:
was this not building before? this should have been an error during tsp compile .

Reply:
ok yeah seems like we have some bug in the compiler I think

filed https://github.com/microsoft/typespec/issues/2059